packages/pigeon/lib/cpp_generator.dart

```diff
@@ -588,7 +588,7 @@ const flutter::StandardMessageCodec& ${api.name}::GetCodec() {
                     indent.write('if ($encodableArgName.IsNull()) ');
                     indent.scoped('{', '}', () {
                       indent.writeln(
-                          'wrapped.emplace(flutter::EncodableValue("${Keys.error}"), WrapError("$argName unexpectedly null."));');
+                          'wrapped = WrapError("$argName unexpectedly null.");');
                       indent.writeln('reply(wrapped);');
```

Comment:
Optional: We don't need to set `wrapped` since we're not getting to the later combined code, so these two lines can collapse to just `return(WrapError("$argName unexpectedly null."));`